### PR TITLE
fix: add debtLimit for addVaultType

### PR DESCRIPTION
**Pull request by turadg**
Fix for https://github.com/Agoric/agoric-sdk/pull/4948#issuecomment-1083649895

### CODE REVIEW

File: loadgen/contract/agent-prepare-loadgen.js [hunk 1]
```diff
@@ -391,6 +391,7 @@ export default async function startAgent({
       ]);
 
       const rates = {
+        debtLimit: AmountMath.make(centralBrand, 1_000_000_000n),
```
Comment (mhofman) [inline loadgen/contract/agent-prepare-loadgen.js:394]: Is this backwards compatible? Aka would having this property provided on revisions before https://github.com/Agoric/agoric-sdk/pull/4948 cause the call to fail?

**Review by mhofman — APPROVED** [review]
LGTM, as long as this change is backwards compatible.

**Comment by turadg** [comment]
Open to ideas on how to avoid breaking in the future. One option is to export a test-utils package from agoric-sdk and have a function that makes test contracts.

File: loadgen/contract/agent-prepare-loadgen.js (hunk 1 above)
Comment (turadg) [reply loadgen/contract/agent-prepare-loadgen.js:394]: It is. The function reading this object isn't concerned if there are other keys it isn't interested in.

**Pull request merged**